BadHabbit/BadHabbit/UserStories/SecondScreen/SecondScreenViewModel.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  SecondScreenViewModel.swift
+//  BadHabbit
+//
+//  Created by Halina Kurylchykava on 25.05.23.
+//
+
+import Foundation
+//typealias Observable<T> = (T) -> Void
```

Comment:
With commented Observable it shouldn't compiled

Reply:
typealias Observable<T>  declared in ExampleViewModel.swift
It will be redeclaration
I deleted this line.